fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/util/SumoRouteFileCreator.java

```diff
@@ -323,7 +300,15 @@ public SumoRouteFileCreator addVehicle(long time, String vehicleId, String vehic
      */
     public void store(File target) {
         // write route-file
-        writeXmlFile(target, routeFileDocument);
+        writeXmlFile(target, vehicleTypesDocument);
+    }
+
+    /**
+     * Stores the document to the given target file.
+     */
+    public void storeDepartures(File target) {
```

Comment:
Exporting all departures was indeed only a debug feature. This way we could debug the actual SUMO traffic generated by the mapping without using MOSAIC.  However, since now routes are not part of the file anymore, this gets quite useless. So I would propose that we remove this debug feature now (also the flag from CSumo).

Reply:
I like 👍